### PR TITLE
Simple alert type editing and removing has_variables

### PR DESCRIPTION
**Pull request by jamesjefferies**

### CODE REVIEW

File: app/views/admin/alert_types/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<h1>Edit: <%= @alert_type.title %> alert type</h1>
+
+<%= simple_form_for(@alert_type, url: admin_alert_type_path(@alert_type)) do |f| %>
+
+  <%= render 'shared/errors', subject: @alert_type, subject_name: 'alert type' %>
+
+  <div class="form-group">
```
Comment (jamesalmond) [inline app/views/admin/alert_types/edit.html.erb]: Do we need these wrapper divs with simple_form?

File: app/views/admin/alert_types/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <%= f.input :title, as: :string, class: 'form-control' %>
```
Comment (jamesalmond) [inline app/views/admin/alert_types/edit.html.erb]: Same with these classes

File: app/views/admin/alert_types/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="form-group">
+    <%= f.input :description, class: 'form-control' %>
+  </div>
+
+  <div class="form-group">
+    <%= f.input :analysis, class: 'form-control' %>
+  </div>
+
+  <div class="form-group">
+    <% AlertType.frequencies.keys.each do |frequency| %>
```
Comment (jamesalmond) [inline app/views/admin/alert_types/edit.html.erb]: What stops us doing something like this here? https://github.com/BathHacked/energy-sparks/blob/master/app/views/onboarding/meters/_form.html.erb#L10

File: app/views/admin/alert_types/show.html.erb [hunk 4]
```diff
@@ -8,14 +8,19 @@
   <dd class="col-md-10"><%= @alert_type.frequency.humanize %></dd>
 
   <dt class="col-md-2">Show ratings</dt>
-  <dd class="col-md-10"><%= @alert_type.show_ratings %></dd>
```
Comment (jamesalmond) [inline app/views/admin/alert_types/show.html.erb]: I don't think we use `show_ratings` for anything. Can we remove it too?

File: app/views/admin/alert_types/edit.html.erb (hunk 1 above)
Comment (jamesjefferies) [reply app/views/admin/alert_types/edit.html.erb]: *cough* no.

File: app/views/admin/alert_types/edit.html.erb (hunk 3 above)
Comment (jamesjefferies) [reply app/views/admin/alert_types/edit.html.erb]: Brilliant, nothing, I'd just forgotten about the label method thingy...

File: app/views/admin/alert_types/show.html.erb (hunk 4 above)
Comment (jamesjefferies) [reply app/views/admin/alert_types/show.html.erb]: Yes, we can

**Pull request merged**